internal/trainings/trainings.go

```diff
@@ -1,13 +1,71 @@
 package trainings
 
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/Yandex-Practicum/tracker/internal/personaldata"
+	"github.com/Yandex-Practicum/tracker/internal/spentenergy"
+)
+
 type Training struct {
-	// TODO: добавить поля
+	Steps        int
+	TrainingType string
+	Duration     time.Duration
+	personaldata.Personal
 }
 
 func (t *Training) Parse(datastring string) (err error) {
-	// TODO: реализовать функцию
+
+	splitstring := strings.Split(datastring, ",")
+
+	if len(splitstring) != 3 {
+		return fmt.Errorf("неверный формат данных")
+	}
+
+	steps, err1 := strconv.Atoi(splitstring[0])
+	if err1 != nil {
+		return fmt.Errorf("ошибка конвертации")
```

Comment:
1. Теряешь сообщение об ошибки
2. Ошибку лучше писать на английском
```suggestion
		return fmt.Errorf("conversion error: %w", err)
```